Code/ImageTransforms.py

```diff
@@ -0,0 +1,77 @@
+import cv2
+import numpy as np
+
+
+# assumes input is a PIL Image
+class PILToNumpy:
+    def __call__(self, image):
+        return np.array(image)
+
+
+# assumes input data is a numpy array
+# should operate on Greyscale image
+class EqualizeHistogram:
+    def __call__(self, image):
+        return cv2.equalizeHist(image)
+
+
+# assumes input data is a numpy array
+# should operate on Grayscale image
+class Gray2RGB:
```

Comment:
It's weird to me that this improves the results.  Given that the images are grayscale, I don't see why this would help, and I could see it hurting.  But if it actually helps .. I won't complain

Reply:
Gray to RGB is required because the densenet121 architecture provided by torchvision is hardcoded to take 3 channels initially and adjusting/modifying that isn't trivial. 

Had a very good discussion with @kshitijbhat330 about it a few days ago. Thought you were on the call... but maybe not. I had looked into potentially changing the densenet model to take only a single channel at the first layer, but upon further digging it isn't something that the torchvision implementation made flexible. 
https://pytorch.org/docs/stable/_modules/torchvision/models/densenet.html#densenet121